xcell/tests/test_cls.py

```diff
@@ -167,6 +167,16 @@ def test_custom_auto():
 
     assert np.allclose(cl1, cl2-np.pi*1E-5, rtol=1E-4, atol=0)
 
+    # Covariance custom cross
```

Comment:
Missing test for the new parts of the method and the new output of the cl.npz file.

Reply:
Aren't these being tested in https://github.com/xC-ell/xCell/blob/ed32e21154b7e85aa10f169b0eb8c970cf99cf7d/xcell/tests/test_mapper_planck_all.py#L84?
What do you want to test about the `npz` files?

Reply:
Done